cql3/lists.cc

```diff
@@ -177,6 +181,10 @@ lists::marker::bind(const query_options& options) {
     }
 }
 
+expr::expression lists::marker::to_expression() {
+    throw std::runtime_error(fmt::format("to_expression not implemented! {}:{}", __FILE__, __LINE__));
```

Comment:
on_internal_error() is nicer, and so is std::source_location(), but I won't insist since this is all temporary code.